src/main/java/ch/wisv/areafiftylan/security/SecurityConfiguration.java

```diff
@@ -123,7 +123,7 @@ public WebMvcConfigurer corsConfigurer() {
         return new WebMvcConfigurer() {
             @Override
             public void addCorsMappings(@Nonnull CorsRegistry registry) {
-                registry.addMapping("/**").exposedHeaders("X-Auth-Token");
+                registry.addMapping("/**").exposedHeaders("X-Auth-Token").allowedMethods("GET", "POST", "PUT", "OPTIONS", "HEAD");
```

Comment:
```suggestion
                registry.addMapping("/**").exposedHeaders("X-Auth-Token").allowedMethods("*");
```

Reply:
I wasn't sure whether this would cause any security concerns, but if you don't think so, I'll change it!

Reply:
No, this does not change any actual security policies of the server, just requests from other domains (cross origin). As main site (https://areafiftylan.nl) is on a different origin than the api (https://api.areafiftylan.nl) we are required to add this for the site to work properly.